src/goto-programs/cfg.h

```diff
@@ -102,10 +102,25 @@ class cfg_baset:public grapht< cfg_base_nodet<T, I> >
 
       return e.first->second;
     }
+
+  private:
+    friend cfg_baset<T, P, I, simp>;
+
+    grapht< cfg_base_nodet<T, I> > &container;
```

Comment:
May be intentional,  but note the spacing avoiding `>>` is not necessary with C++11 onwards